uiowa/uiowa
# Research articles migration

Relates to https://github.com/uiowa/uiowa/issues/5248




# Todo

- [x] Target the correct content type...
- [x] Redirects
- [x] Article type if external or ianow, link directly to source
- [x] Title, Body, Teaser, Summary, other basic Article fields.
- [x] Byline to Organization
- [x] Featured image
- [x] Do not display featured image for all migrated
- [x] Contacts mapping - https://github.com/uiowa/uiowa/issues/5248#issuecomment-1341387260

# How to test

```
ddev blt ds --site=research.uiowa.edu && ddev drush @research.local en research_migrate -y && ddev drush @research.local uli /admin/config/sitenow/sitenow-migrate
```

Complete the form following the help text
`ddev drush @research.local ms`
`ddev drush @research.local mim --all`

Check any migration errors and that the redirects are working. Review the migration deliverables here https://github.com/uiowa/uiowa/issues/5248, particularly https://github.com/uiowa/uiowa/issues/5248#issuecomment-1341387260

Note: there does seem to be a https://research.uiowa.ddev.site/contact-us/cheryl-ridgeway but no migration content is linked to this. Notes say to delete. Could just delete this one manually.
